nginx/datadog_checks/nginx/nginx.py

```diff
@@ -131,8 +131,9 @@ def _perform_service_check(self, instance, url, ssl_validation, auth):
         parsed_url = urlparse.urlparse(url)
         nginx_host = parsed_url.hostname
         nginx_port = parsed_url.port or 80
+        custom_tags = instance.get('tags', [])
```

Comment:
`custom_tags` can be `None`

Reply:
if `custom_tags` is None, won't the `get(...)` set `custom_tags=[]`?